scripts/make_release.sh

```diff
@@ -0,0 +1,4 @@
+./scripts/uninstall.sh
+./scripts/clean_up.sh
+python setup.py register
+python setup.py sdist bdist_wheel upload
```

Comment:
@barseghyanartur I would strongly suggest you to take a look at zest.releaser, is as simple as:

```
source bin/activate
pip install -U zest.releaser[recommended]
fullrelease
```